images/macos/tests/Common.Tests.ps1

```diff
@@ -149,6 +149,16 @@ Describe "CodeQL" -Skip:($os.IsCatalina) {
         $CodeQLPacksPath = Join-Path $CodeQLVersionPath -ChildPath "x64" | Join-Path -ChildPath "codeql" | Join-Path -ChildPath "qlpacks"
         $CodeQLPacksPath | Should -Exist
     }
+
+    It "prior codeql" {
+        $PriorCodeQLVersionsWildcard = Join-Path $Env:AGENT_TOOLSDIRECTORY -ChildPath "CodeQL" | Join-Path -ChildPath "*"
+        $PriorCodeQLVersionPath = Get-ChildItem $PriorCodeQLVersionsWildcard | Select-Object -Last 1 -Expand FullName
+        $PriorCodeQLPath = Join-Path $PriorCodeQLVersionPath -ChildPath "x64" | Join-Path -ChildPath "codeql" | Join-Path -ChildPath "codeql"
+        "$PriorCodeQLPath version --quiet" | Should -ReturnZeroExitCode

```

Comment:
Optional: We could potentially have a test that the current and prior versions of CodeQL are different.

Reply:
I've added this test separately. It does add additional time in now calling `codeql version` two more times (so four total calls) for the extra test in order to compare the versions to one another. 

Alternatively I could collapse all three tests into one that would together test that the versions are different, and that each pack path exists, which would decrease the amount of repeated work/calls to `codeql version`. What do you think?

Reply:
I think repeated calls to `codeql version` are probably fine, and we should do whatever's most readable.  But it probably makes sense to do this as a followup rather than in this PR.

Reply:
Sounds good — I will leave this thread unresolved so I remember to do the followup.